internal/git/worktree_test.go

```
@@ -0,0 +1,84 @@
+	if err != nil {
+		t.Fatal(err)
+	}
+	if string(contents) != "base\n" {
+		t.Fatalf("worktree README = %q, want target branch contents", contents)
+	}
+}
+
+// runGit executes one Git command in a temporary test repository.
+func runGit(t *testing.T, directory string, args ...string) string {
+	t.Helper()
+	command := exec.Command("git", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)